### ISSUE TITLE
Crash on android

### ISSUE THREAD

**Issue by wirekang** [AlwaysLoveme/capacitor-plugin-safe-area#4]
Step to reproduce: Launch app

Error log:
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: __PACKAGE__, PID: 20067
    java.lang.RuntimeException: Unable to resume activity {__PACKAGE__/__PACKAGE__.MainActivity}: java.lang.ClassCastException: com.zhuxian.safearea.SafeAreaPlugin cannot be cast to android.hardware.SensorEventListener
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5230)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5269)
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54)
        at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:226)
        at android.os.Looper.loop(Looper.java:313)
        at android.app.ActivityThread.main(ActivityThread.java:8669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
     Caused by: java.lang.ClassCastException: com.zhuxian.safearea.SafeAreaPlugin cannot be cast to android.hardware.SensorEventListener
        at com.zhuxian.safearea.SafeAreaPlugin.handleOnResume(SafeAreaPlugin.java:44)
        at com.getcapacitor.Bridge.onResume(Bridge.java:1191)
        at com.getcapacitor.BridgeActivity.onResume(BridgeActivity.java:86)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1476)
        at android.app.Activity.performResume(Activity.java:8441)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5200)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5269) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54) 
        at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:226) 
        at android.os.Looper.loop(Looper.java:313) 
        at android.app.ActivityThread.main(ActivityThread.java:8669) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135) 
```

OS:
```
Device: Samsung Galaxy S22
Android: 12
```

**Comment by AlwaysLoveme** [comment]
> Step to reproduce: Launch app
> 
> Error log:
> 
> ```
> E/AndroidRuntime: FATAL EXCEPTION: main
>     Process: __PACKAGE__, PID: 20067
>     java.lang.RuntimeException: Unable to resume activity {__PACKAGE__/__PACKAGE__.MainActivity}: java.lang.ClassCastException: com.zhuxian.safearea.SafeAreaPlugin cannot be cast to android.hardware.SensorEventListener
>         at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5230)
>         at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5269)
>         at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54)
>         at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
>         at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
>         at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
>         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438)
>         at android.os.Handler.dispatchMessage(Handler.java:106)
>         at android.os.Looper.loopOnce(Looper.java:226)
>         at android.os.Looper.loop(Looper.java:313)
>         at android.app.ActivityThread.main(ActivityThread.java:8669)
>         at java.lang.reflect.Method.invoke(Native Method)
>         at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
>         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
>      Caused by: java.lang.ClassCastException: com.zhuxian.safearea.SafeAreaPlugin cannot be cast to android.hardware.SensorEventListener
>         at com.zhuxian.safearea.SafeAreaPlugin.handleOnResume(SafeAreaPlugin.java:44)
>         at com.getcapacitor.Bridge.onResume(Bridge.java:1191)
>         at com.getcapacitor.BridgeActivity.onResume(BridgeActivity.java:86)
>         at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1476)
>         at android.app.Activity.performResume(Activity.java:8441)
>         at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5200)
>         at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5269) 
>         at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54) 
>         at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45) 
>         at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
>         at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
>         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2438) 
>         at android.os.Handler.dispatchMessage(Handler.java:106) 
>         at android.os.Looper.loopOnce(Looper.java:226) 
>         at android.os.Looper.loop(Looper.java:313) 
>         at android.app.ActivityThread.main(ActivityThread.java:8669) 
>         at java.lang.reflect.Method.invoke(Native Method) 
>         at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571) 
>         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135) 
> ```
> 
> OS:
> 
> ```
> Device: Samsung Galaxy S22
> Android: 12
> ```

Hi, fixed in version 0.0.7

**Comment by digaus** [comment]
See: https://github.com/AlwaysLoveme/capacitor-plugin-safe-area/issues/3#issuecomment-1230333168
